packages/blocks/src/api/raw-handling/iframe-remover.js

```diff
@@ -15,3 +15,13 @@ export default function( node ) {
 		remove( node );
 	}
 }
+
+export function checkForUntransformedIframe( rawTransform, node ) {
+	const frameRegex = /(?:<iframe[^>]*)(?:(?:\/>)|(?:>.*?<\/iframe>))/gi;
+	return (
+		rawTransform &&
+		node.nodeName === 'FIGURE' &&
```

Comment:
Just trying to work out why iframes get wrapped in a figure node, as this doesn't happen if you manually pass an iframe trough the figure-content-reducer

Reply:
Looks like this is intended behaviour as most other embedded content also appears to be wrapped in <figure> nodes